test/unit/test_lapack_tile/test_potrf.h

```diff
@@ -90,14 +90,14 @@ void testPotrf(blas::Uplo uplo, SizeType n, SizeType extra_lda) {
 
 template <class T, bool return_info>
 void testPotrfArgExceptions(blas::Uplo uplo, TileElementSize size_a, SizeType extra_lda) {
-  SizeType lda = std::max(1, size_a.rows()) + extra_lda;
+  SizeType lda = std::max<SizeType>(1, size_a.rows()) + extra_lda;
 
   std::stringstream s;
   s << "POTRF Arguments Exceptions: " << uplo;
   s << ", size_a = " << size_a << ", lda = " << lda;
   SCOPED_TRACE(s.str());
 
-  memory::MemoryView<T, Device::CPU> mem_a(lda * size_a.cols());
+  memory::MemoryView<T, Device::CPU> mem_a(util::size_t::mul(lda, size_a.cols()));
```

Comment:
There are 3 things here:
- If `SizeType` is to have `operator*()`, I will use that instead of `util::size_t::mul()`, it's easier to read
- the namespace `size_t` is rather confusing because there is a commonly used `std` type called `std::size_t`  already
- I'd encourage heavier use of `using` declarations to shorten these expressions and improve readability: e.g. `using util::size_t::mul;` or `using MemViewT = memory::MemoryView<T, Device::CPU>`, etc.

Reply:
size_t::mul is different from operator*():
`mul` takes the operands, convert them to std::size_t and multiply them.
(The result may not fit in SizeType and the result has to be converted to size_t anyway.)

Regarding the 3rd point, for size_t add and mul `using` is generally used.
I agree we can use it more, but we should still pay attention that using it too much can be worse. I worked once with a code that had tens of using statement and identifying a type was a nightmare.

Reply:
Starting from the last, I'm going to address your suggestion.

About first two points, it is worth sharing with you a bit of context.
`MemoryView` constructor accepts as parameter the dimension of the memory area, and I think is correct to use `std::size_t`.

The function `dlaf::util::size_t::mul` is an helper function that allows us to (quite) safely compute the multiplication
https://github.com/eth-cscs/DLA-Future/blob/1922140d1bca8f9448f2556393d5961cc1d098ab/include/dlaf/util_math.h#L77-L84

As you can see, the name of the namespace is strictly related to the fact that it is a group of helper functions for `std::size_t` arithmetic.

Reply:
> I agree we can use it more, but we should still pay attention that using it too much can be worse. I worked once with a code that had tens of using statement and identifying a type was a nightmare.

As most all features, it shouldn't be taken to the extreme. I presume the code you mention utilized type aliases heavily, that indeed may cause trouble with identifying types. I would say that, if you are at the point that you have so many using statements and types in the same function, it is perhaps reasonable to consider refactoring into smaller functions.

> About first two points, it is worth sharing with you a bit of context.
MemoryView constructor accepts as parameter the dimension of the memory area, and I think is correct to use std::size_t.
> 
> The function dlaf::util::size_t::mul is an helper function that allows us to (quite) safely compute the multiplication

> mul takes the operands, convert them to std::size_t and multiply them.

Sure thing! You can ignore the first and second bullet points.